src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -197,6 +197,13 @@
                     </Setter.Value>
                 </Setter>
             </Style>
+
+            <Style x:Key="TopLibraryTreeViewItem"
```

Comment:
Added this `Style` to get rid of this situation:
![image](https://cloud.githubusercontent.com/assets/8158404/9109984/98f7b9f0-3c42-11e5-87b9-a9583ad099de.png) ![image](https://cloud.githubusercontent.com/assets/8158404/9109995/bb826272-3c42-11e5-869b-f527ebcf9177.png)

Now top result is always highlighted. Maybe, @Racel will find this fix wrong. In that case I will need to create a copy of top result node, so that they will use different ViewModels. (and 1 will be highlighted, 2 won't be highlighted)